### ISSUE TITLE
CKS instance Info page display all Public IP's PNAT,FW and LB in same account.

### ISSUE THREAD

**Issue by levindecaro** [apache/cloudstack#6133]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI

~~~

##### CLOUDSTACK VERSION

~~~
4.16.1
~~~

##### SUMMARY

After provision a CKS instance, navigate to the CKS info page, Firewall,Port Forwarding and Load Balancing are showing all existing rule from all Public IP address within same account.

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~
Should display Public IP rules associated for CKS instead of all IP.
~~~

##### ACTUAL RESULTS

CKS info page are showing all existing rules of other Public IP information.

<img width="1186" alt="image" src="https://user-images.githubusercontent.com/36956864/158811009-6deb7180-c3c9-4e35-8039-57fb1f71cd7e.png">

<img width="1183" alt="image" src="https://user-images.githubusercontent.com/36956864/158810982-7f6c70d0-459a-423e-a5dc-5899bb470754.png">

<img width="1185" alt="image" src="https://user-images.githubusercontent.com/36956864/158810950-28167b60-520a-45bb-8a4f-07f3ba694367.png">

Public IP Page of CKS are showing correct information.

<img width="1173" alt="image" src="https://user-images.githubusercontent.com/36956864/158811314-b68b19a4-2145-4dc5-b41f-5f110e5b3997.png">

<img width="1169" alt="image" src="https://user-images.githubusercontent.com/36956864/158811348-fa511122-9770-44e2-b88b-75fb61d90e02.png">

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/36956864/158811381-e1b6b29a-aeae-417a-ab1a-1880bf7805da.png">

~~~

~~~

**Comment by nvazquez** [comment]
Hi @davidjumani can you please check this issue?

**Comment by davidjumani** [comment]
@levindecaro Can you check the API call being made when listing the firewall / LB / PF rules in the kubernetes tab ? I'm unable to reproduce the issue

**Comment by levindecaro** [comment]
@davidjumani I only have apilog.log on hand at this moment, is it helpful for you? Btw, I re-test few times on that, found this is happening in the middle of status change, like starting the cluster or stopping the cluster, once the state become started/stopped, it will show correct rules.

**Comment by davidjumani** [comment]
@levindecaro That helps, thanks. I'll have a look into it

**Comment by levindecaro** [comment]
@davidjumani  thank you

[13.log](https://github.com/apache/cloudstack/files/8348541/13.log)

**Comment by nvazquez** [comment]
@Pearl1594 can you please check this issue?

**Comment by weizhouapache** [comment]
@levindecaro 
I cannot reproduce the issue on latest main branch.

fixed an issue with listing LB rules: #7058

**Comment by DaanHoogland** [comment]
@levindecaro I am closing this issue. If you feel this is invalid please reopen or create a new one.
